docs/api/v4/changes.md

```diff
@@ -4,6 +4,14 @@
 
 Also see: [Differences Between Api v3 and v4](../v4/differences-with-v3.md) and [Hooks Changelog](../../hooks/changes.md).
 
+## CiviCRM 5.29
+
+### 5.29.0
+
+Added `target_contact_id` and `assignee_contact_id` to `Activity.create` in the API explorer. Each can take an array of contact IDs. Nb. this feature existed before 5.29.0 but was previously not discoverable in the explorer.
+
+See https://github.com/civicrm/civicrm-core/commit/8c6a5fd64b
```

Comment:
```suggestion
For more information see the [commit to CiviCRM Core](https://github.com/civicrm/civicrm-core/commit/8c6a5fd64b)
```

Reply:
Sure, that will change gradually :-) "bare" links are a violation of the common mark standard so we should avoid them if possible!